runtime/doc/api.txt

```diff
@@ -1307,6 +1307,34 @@ nvim_put({lines}, {type}, {after}, {follow})                      *nvim_put()*
                   |P|).
       • {follow}  If true place cursor at end of inserted text.
 
+nvim_redraw({opts})                                            *nvim_redraw()*
+    Instruct Nvim to redraw various elements.
+
+    Parameters: ~
+      • {opts}  Optional parameters.
+                • flush: Update the screen with pending updates, even if Nvim
+                  is waiting for input and would normally not do so.
+                • clear: Forcefully redraw `win`, all windows containing
+                  `buf`, or all windows when both are omitted.
+                • range: Buffer range to redraw, useful for decoration
+                  providers. Tuple `[first, last]`:
+                  • first: first line number of the range
+                  • last: last line number of the range
+                • cursor: Immediately update cursor position on the screen.
+                • status: Redraws (a combination of) the following status
+                  components (array of one or more strings):
+                  • "bar": |'winbar'|
+                  • "column": |'statuscolumn'|
+                  • "line": |'statusline'|
+                  • "tab": |'tabline'|
```

Comment:
I don't think adding these new abbreviations is a good idea. I think it's better to use existing terminologies.

Reply:
I guess it's better to put them directly in `opts`, so that there will be fewer thing to decide when something new need to be added.